specification/consumption/resource-manager/Microsoft.Consumption/stable/2021-05-01/consumption.json

```diff
@@ -5089,6 +5154,14 @@
         "status": {
           "description": "The status of the lot.",
           "type": "string",
+          "enum": [
+            "None",
+            "Active",
+            "Inactive",
+            "Expired",
+            "Complete",
+            "Canceled"
+          ],
```

Comment:
The following needs to be added

"x-ms-enum": {
            "name": "Status",
            "modelAsString": true
          }

Reply:
Added.